lib/matplotlib/contour.pyi

```diff
@@ -113,6 +112,8 @@ class ContourSet(ContourLabeler, Collection):
     @property
     def alpha(self) -> float | None: ...
     @property
+    def antialiased(self) -> bool: ...
```

Comment:
setter should be added to the pyi as well

Reply:
Done